test/studies/isx/isx-no-return.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: total =, bucket count =, bucket offset =, bucketize =, exchange =, count keys =
+files: isx-no-return.dat
```

Comment:
Huh.. I have no idea how that's working.. I would still change them to repeat-files